docs/pureconfig.md

```diff
@@ -0,0 +1,27 @@
+# Module PureConfig
+
+![GitHub tag (latest SemVer)](https://img.shields.io/github/v/tag/avast/scala-server-toolkit?label=version&sort=semver)
+
+`libraryDependencies += "com.avast.server.toolkit" %% "scala-server-toolkit-pureconfig" % "<VERSION>"`
+
+This module allows you to load your application's configuration file according to a case class provided to it. It uses
+[PureConfig](https://pureconfig.github.io) library to do so which uses [Lightbend Config](https://github.com/lightbend/config) which means
+that your application's configuration will be in [HOCON](https://github.com/lightbend/config/blob/master/HOCON.md) format.
+
```

Comment:
Could you please write the rationale why this module is useful here? If I understand it correctly, `ConfigSource.default.load[A]` is an operation with side-effect and so it should be wrapped in an `F`. And this is what this module is actually fixing, right?

Reply:
OK